src/physics/utils/pio_reader.F90

```diff
@@ -1843,8 +1335,8 @@ subroutine get_netcdf_var_char_0d(this, varname, var, errmsg, errcode)
          return
       end if
 
-      !Get variable type and number of variable dimensions on file:
-      errcode = pio_inquire_variable(pio_file_handle, var_id, xtype=nc_type, ndims=ndims)
+      !Get variable type  on file:
+      errcode = pio_inquire_variable(pio_file_handle, var_id, xtype=nc_type)
```

Comment:
question that's not exactly related to these changes:

why do we not need to check the variable type for other interfaces (int, real)? Will PIO handle those situations gracefully?